okta-workitem/pom.xml

```diff
@@ -0,0 +1,185 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jbpm.contrib</groupId>
+    <artifactId>workitems</artifactId>
+    <version>7.16.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>okta-workitem</artifactId>
+  <name>Okta</name>
+  <description>Interact with Okta management API</description>
+  <organization>
+    <name>jBPM team</name>
+  </organization>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.okta.sdk</groupId>
+      <artifactId>okta-sdk-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.okta.sdk</groupId>
+      <artifactId>okta-sdk-impl</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.okta.sdk</groupId>
+      <artifactId>okta-sdk-httpclient</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.kie</groupId>
+      <artifactId>kie-api</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.kie</groupId>
+      <artifactId>kie-internal</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>jbpm-workitems-core</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>jbpm-document</artifactId>
```

Comment:
this does not seem to be used at all in these work item handlers

Reply:
updated - thanks!